src/developer-testing/developer-testing.module.ts

```diff
@@ -0,0 +1,17 @@
+import { Module } from '@nestjs/common';
```

Comment:
I talked to Jere about this, and we were thinking this would be nice to have in to help testing out the feature.

I added in a flag so it won't show up in the docs normally.